docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+      {toolbar}
+      {actionBar}
+      <PickersLayoutContentWrapper className={pickersLayout.contentWrapper}>
+        {tabs}
+        {content}
+      </PickersLayoutContentWrapper>
+    </PickersLayoutRoot>
+  );
+}
+export default function CustomStaticDatePicker() {
+  return <StaticDatePicker slots={{ layout: MyCustomLayout }} />;
+}
+```
+
+#### [Fields: the new default input for Pickers](/x/react-date-pickers/fields/)
+
+These rich text fields are specialized for date and time logic and offer quick navigation and isolated interaction within each section of a date value.
+
+<video style="margin-bottom: 5px; width: 608px;" autoplay muted loop playsinline >
+  <source src="/static/blog/v6-beta-pickers/date-field-navigation.mp4" type="video/mp4" />
+</video>
+
+#### [Shortcuts for picking specific dates in a calendar](/x/react-date-pickers/shortcuts/)
+
+Add quick and customizable shortcuts for your users. Particularly useful for date ranges.
+Display them on the left, right, bottom, or top.
+
+<img src="/static/blog/v6-beta-pickers/date-range-shortcuts.png" style="width: 692px; margin-bottom: 24px;" loading="lazy" alt="Date Range shortcuts." />
+
+#### [Edit date ranges with drag and drop](/x/react-date-pickers/date-range-calendar/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+Editing a date range is even easier now with the new drag-and-drop interface. Change `start` and `end` dates at will.
+
+<video style="margin-bottom: 24px; max-height: 398px;" autoplay muted loop playsinline >
+    <source src="/static/blog/v6-beta-pickers/edit-drag.mp4" type="video/mp4" />
+</video>
+
+#### Removed clock view on desktop Time Pickers
+
+Many users complained, and we completely agree, the clock view is not the ideal time-picking experience on Desktop, so we removed it as a default view.
+You can still use the Clock if you like to, but a replacement UI for the time picker is coming shortly after this release.
+
+## Installation and migration
+
+If this is your first try with MUI X, you can jump in the getting started section:
+
+- [Data Grid](/x/react-data-grid/getting-started/)
+- [Date Pickers](/x/react-date-pickers/getting-started/)
+
+If you're coming from previous versions, we recommend you check our migration guide with the complete list of breaking changes:
+
+- [Data Grid](/x/migration/migration-data-grid-v5/)
+- [Date Pickers](/x/migration/migration-pickers-v5/)
+
+We also provide codemods to automate some of the necessary updates in your codebase.
+
+```sh
+npx @mui/x-codemod v6.0.0/preset-safe <path>
+```
+
+## What's next?
+
+We operate in a continuous-delivery environment, so the plan is to keep rolling out new features in minor versions.
+
+With the new base established in v6.0.0, we'll continue our efforts to improve UI/UX, bring more customization abilities, and support new use cases.
+
+Here's a list of what you can expect to be delivered in the following months.
```

Comment:
Reading it, I felt that it could help to precise that we don't plan to stop at this list.

```suggestion
Here's a list of what you can expect to be delivered in the following months (what is next on our immediate roadmap).
```